capstone-frontend/src/ClubGridItem.test.js

```diff
@@ -16,7 +16,8 @@ const club = {
   name: "name",
   description: "description",
   gbook: "gbook", 
-  memberIDs: ["one", "two", "three"]
+  memberIDs: ["one", "two", "three"], 
+  requestIDs: ["four", "five"]
```

Comment:
nit: Add a trailing comma here so this doesn't get caught in future diffs like the line above when you add new properties (https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma).

Reply:
Didn't know that, but makes a lot of sense. Will change.